### PR TITLE
datestyle workaround - https://github.com/r-dbi/RPostgres/issues/287

### PR DESCRIPTION
**Pull request by baderstine**
add datestyle workaround in dbConnect, closes https://github.com/r-dbi/RPostgres/issues/287

### CODE REVIEW

File: R/PqConnection.R [hunk 1]
```diff
@@ -205,6 +205,9 @@ setMethod("dbConnect", "PqDriver",
     )
     on.exit(dbDisconnect(conn))
 
+    # set datestyle workaround - https://github.com/r-dbi/RPostgres/issues/287
+    dbExecute(conn, "SET datestyle to iso, mdy;")
```
Comment (krlmlr) [inline R/PqConnection.R:209]: The semicolon feels unnecessary.

```suggestion
    dbExecute(conn, "SET datestyle to iso, mdy")
```

**Review by krlmlr — COMMENTED** [review]
Thanks for looking into it!

File: R/PqConnection.R (hunk 1 above)
Comment (baderstine) [reply R/PqConnection.R]: removed in latest commit.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**